<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyUni2Kor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon-128.png</normaloff>icon-128.png</iconset>
  </property>
  <widget class="QCheckBox" name="checkBox_alwaysOnTop">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Always on top</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_uni">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Unicode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Korean</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_uni">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_kor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
